tests/nnfw_api/src/one_op_tests/Cast.cc

```diff
@@ -89,6 +89,34 @@ TEST_F(GenModelTest, OneOp_Cast_BoolToInt32)
   SUCCEED();
 }
 
+TEST_F(GenModelTest, OneOp_Cast_Uint8ToFloat32)
+{
+  CircleGen cgen = genSimpleCastModel(circle::TensorType_UINT8, circle::TensorType_FLOAT32);
+
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  // clang-format off
+  _context->addTestCase(
+    TestCaseData{}.addInput<uint8_t>({0, 100, 200, 255})
+                  .addOutput<float>({0., 100., 200., 255.}));
+  // clang-format on
+  _context->setBackends({"acl_cl", "acl_neon", "cpu"});
+
+  SUCCEED();
+}
+
+TEST_F(GenModelTest, OneOp_Cast_Int64ToFloat32)
+{
+  CircleGen cgen = genSimpleCastModel(circle::TensorType_INT64, circle::TensorType_FLOAT32);
+
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  _context->addTestCase(TestCaseData{}
+                          .addInput<int64_t>({-12345, 3, 100, 2147483648})
+                          .addOutput<float>({-12345., 3., 100., 2147483648.}));
+  _context->setBackends({"acl_cl", "acl_neon", "cpu"});
```

Comment:
```suggestion
  _context->setBackends({"acl_cl", "cpu"});
```

Reply:
@chunseoklee I've removed `acl-neon` as your suggestion. May I ask why int64 case should be disabled for acl-neon?

Reply:
I am not 100% sure, but here is the fail log : 
```
[2022-04-19T11:08:57.516Z] [ RUN      ] GenModelTest.OneOp_Cast_Int64ToFloat32
[2022-04-19T11:08:57.516Z] Error during nnfw_session::run : in run src/core/NEON/kernels/NEDepthConvertLayerKernel.cpp:1411: Not supported
[2022-04-19T11:08:57.516Z] /opt/jenkins_slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline-release/tests/nnfw_api/src/GenModelTest.h:390: Failure
[2022-04-19T11:08:57.516Z] Expected equality of these values:
[2022-04-19T11:08:57.516Z]   (nnfw_run(_so.session))
[2022-04-19T11:08:57.516Z]     Which is: 1
[2022-04-19T11:08:57.516Z]   NNFW_STATUS_NO_ERROR
[2022-04-19T11:08:57.516Z]     Which is: 0
[2022-04-19T11:08:57.516Z] [  FAILED  ] GenModelTest.OneOp_Cast_Int64ToFloat32 (31 ms)
```

Reply:
@chunseoklee You're right. `acl-cl` also doesn't support S64 type. I will disable int64 test for `acl-cl`, too.
```
[2022-04-19T13:26:32.535Z] Error during model prepare : in validate_arguments src/core/CL/kernels/CLDepthConvertLayerKernel.cpp:55: ITensor data type S64 not supported by this kernel
[2022-04-19T13:26:32.535Z] /opt/jenkins_slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline/tests/nnfw_api/src/GenModelTest.h:319: Failure
[2022-04-19T13:26:32.535Z] Expected equality of these values:
[2022-04-19T13:26:32.535Z]   (nnfw_prepare(_so.session))
[2022-04-19T13:26:32.535Z]     Which is: 1
[2022-04-19T13:26:32.535Z]   NNFW_STATUS_NO_ERROR
[2022-04-19T13:26:32.535Z]     Which is: 0
```